image-prune

```diff
@@ -176,8 +177,15 @@ class S3ImageStore(ImageCache):
             yield datetime.datetime.strptime(stamp, rfc3339ish).timestamp(), name
 
     def delete_file(self, filename):
-        with s3.urlopen(self.url._replace(path='/' + filename), method='DELETE'):
-            pass
+        try:
+            with s3.urlopen(self.url._replace(path='/' + filename), method='DELETE'):
+                pass
+        except urllib.error.HTTPError as e:
+            if e.code == 404:
```

Comment:
Why not do `!= 404: raise` ?

Reply:
What? Elegance? That's not how we do things here! 🙈

Thanks, fixed. (That's what you get when doing this with only half attention)